packages/InteractiveProfilingTool-Core.package/IPTStats.class/instance/formatGarbageCollectionGeneration.withStats..st

```diff
@@ -0,0 +1,10 @@
+formatting
+formatGarbageCollectionGeneration: aLabel withStats: aDict
+
+	| values |
+	
+	values := { #count. #time. #relativeTime. #averageTime } collect: [ :key | aDict at: key ].
+
+	^ ((aDict at: #averageTime) notNil)
```

Comment:
The outmost braces are not needed.

Reply:
Danke für die coole Idee 👍 . Wurde umgesetzt.